docs/CONTRIBUTING.md

```diff
@@ -1,25 +1,46 @@
 # Contributing to Endless Sky
 
-## Posting issues
+## Issues
+
+### Posting issues
 
 The [issues page](https://github.com/endless-sky/endless-sky/issues) on GitHub is for tracking bugs and feature requests. When posting a new issue, please:
 
 * Check to make sure it's not a duplicate of an existing issue.
 * Create a separate "issue" for each bug you are reporting and each feature you are requesting.
+* Choose the correct issue template and fill it out to the best of your ability.
 * Do not use the issues page for things other than bug reports and feature requests. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead.
 
 If requesting a new feature, first ask yourself: will this make the game more fun or interesting? Remember that this is a game, not a simulator. Changes will not be made purely for the sake of realism, especially if they introduce needless complexity or aggravation.
 
-## Posting pull requests
+### Closing issues
+
+If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please open a new issue.
```

Comment:
A more specific bit of text here I think is worth adding. The contrast reading has a slightly increased chance of sticking in people's memories after reading.

```suggestion
If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please do not reopen your issue. Please open a new issue and explain the gap that you feel still needs to be addressed.
```